packages/dom/src/stylo_to_parley.rs

```diff
@@ -98,6 +98,18 @@ pub(crate) fn style(style: &stylo::ComputedValues) -> parley::TextStyle<'static,
         .map(|f| (f * 255.0) as u8);
     let color = peniko::Color { r, g, b, a };
 
+    let decoration_brush = style.get_text()
+            .text_decoration_color
+            .as_absolute()
+            .map(|color| {
+                let [r, g, b, a] = color
+                    .to_color_space(style::color::ColorSpace::Srgb)
+                    .raw_components()
+                    .map(|f| (f * 255.0) as u8);
+                let color = peniko::Color { r, g, b, a };
```

Comment:
Could this color conversion logic be pulled out into a function please? (seeing as it is now duplicated)

Reply:
I've implemented a helper, but it's basically the same as in [/blitz/src/utils.rs](https://github.com/DioxusLabs/blitz/blob/133608dab382b327843f1f1c11753e0cfabc089d/packages/blitz/src/util.rs#L35C2-L57C6) just in dom.

Reply:
Yeah, I'm aware of the duplication but we can't use the code from `blitz` in `blitz-dom`. We might be able to make the dep go the other way. But it's also a small enough amount of code that maybe we can just duplicate it (I'm not anticipating there being a huge amount more code that needs to be duplicated as there is mostly quite a neat layout vs. presentation split - but color happens to be needed in both).